packages/expo-system-ui/package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "expo-system-ui",
+  "version": "1.0.0",
```

I'd start with `0.1.0`, we can bump the major version later when the package is ready and announced
```suggestion
  "version": "0.1.0",
```

---

Technically `0.1.0` is magic zero for `1.0.0`, i.e. react-native. In which case `0.0.0` or `1.0.0-alpha.1` would better represent alpha.